src/main/resources/plugin.yml

```
@@ -14,11 +16,19 @@ commands:
     description: "Gives the player an additional creative plot."
     usage: "/givecreativeplot <player>"
     permission: mythicstore.giveplot
+  skincontrol:
+    description: "Main command for SkinControl."
+  runonjoin:
+    aliases:
+      - roj
 
 permissions:
   mythicstore.reload:
     description: "Reload MythicStore config."
     default: op
   mythicstore.giveplot:
     description: "Give a player a plot"
+    default: op
+  skincontrol.purchased:
```

Better match namespace, we can change the permission in Mythic.
```suggestion
  mythicstore.purchasedskin:
```